spec.emu

```diff
@@ -136,189 +179,307 @@ location: https://github.com/tc39/proposal-decimal/
         1. Return the result of Call ([_argument_].[[equals]], _rounded_).
       </emu-alg>
     </emu-clause>
+    <emu-clause id="sec-decimal128-value-to-object" type="abstract operation">
+      <h1>
+        Decimal128ValueToObject (
+          _argument_ : a Decimal128 value
+        ): a Decimal128 object
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>It converts a Decimal128 value to a Decimal128 object.</dd>
+      </dl>
+      <emu-alg>
+        1. Let _O_ be ! OrdinaryCreateFromConstructor(NewTarget, *"%Decimal128.prototype%"*, « [[Decimal128Data]] »).
```

Comment:
This operation may fail if NewTarget is a user-supplied constructor. However, I don't think we actually want to allow that here.
```suggestion
        1. Let _O_ be ! OrdinaryCreateFromConstructor(%Decimal128%, *"%Decimal128.prototype%"*, « [[Decimal128Data]] »).
```

Reply:
Yeah. I wasn't sure what to do; I just want to construct a fresh Decimal128 value and I didn't think to use `%Decimal128`. Thanks!